### DISCUSSION TITLE
如何获得Galaxy Brain呀？

### DISCUSSION THREAD

**Discussion by yynaixi** [community/community#52588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

如题

**Comment by Sunhaha520** [comment, marked as answer]
需要被认可的回答达到2次。

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
